projects/lib/src/admin/components/asset-list/asset-list.component.html

```diff
@@ -28,17 +28,15 @@ <h1>Assets</h1>
         </div>
       </div>
       <div *ngIf="!loading">
-        <div *ngFor="let asset of assets">
+        <div *ngFor="let asset of assets.Items">
           <cms-asset-update
-            [resourceType]="resourceType"
-            [resourceID]="resourceID"
```

Comment:
Sorry if I wasn't clear enough, but these options should still be available... Winmark just won't be using them.

Reply:
`defaultListOptions` wouldn't work for filtering if they use those two fields (since list by resource doesn't work with filtering)... but if marketplace or some other angular-cms-components user wanted to list by resource assignment I think that option is still useful!

Reply:
also - `beforeAssetUpload` should always get called (if configured) - regardless of if they are using resourceID / Type or not.